test/fuzz/auto-compounder-views/IsCompoundable.fuzz.t.sol

```diff
@@ -0,0 +1,256 @@
+/**
+ * Created by Pragma Labs
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+pragma solidity 0.8.22;
+
+import { AutoCompounder_Fuzz_Test, AutoCompounder } from "../auto-compounder/_AutoCompounder.fuzz.t.sol";
```

Comment:
I prefer to use single imports per line.
Makes it much easier to refactor code (which we unfortunately often have to do).